plugins/aws/s3/src/hopeit/aws/s3/object_storage.py

```diff
@@ -253,8 +257,8 @@ async def get_file(
         """
 
         async with self._session.client(S3, **self._conn_config) as object_storage:
-            if self.partition_dateformat:
-                file_name = f"{self.prefix or ''}{partition_key + '/' if partition_key else ''}{file_name}"
+            file_name = f"{partition_key}/{file_name}" if partition_key else file_name
+            file_name = f"{self.prefix or ''}{file_name}"
```

Comment:
Same recommendation based on previous comment here.

Reply:
I have created https://github.com/hopeit-git/hopeit.aws/issues/6 to address this.